lib/rules/at-rule-disallowed-list/index.js

```diff
@@ -47,6 +47,7 @@ const rule = (primary) => {
 				node: atRule,
 				result,
 				ruleName,
+				word: `@${atRule.name}`,
```

Comment:
Ah, this also can be simplified: 😅 

```suggestion
				word: `@${name}`,
```